spec/post_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative './rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.create(
+      name: 'Loren',
+      photo: 'https://this-person-does-not-exist.com/en/download-page?image=genef4e9868ae582ca3061881b69d8fbeb1',
+      posts_counter: 0,
+      bio: 'I am Loren, and I love sport.'
+    )
+  end
+
+  subject do
+    Post.new(
+      title: 'My first post',
+      author: user,
+      text: 'The motivation to become developer is very big.',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+
+  before { subject.save }
+
```

Comment:
- [ ] It would be better if we implement test for title exceeding 250 characters as seen in the requirements
![Screenshot from 2023-05-05 13-26-53](https://user-images.githubusercontent.com/46388113/236468853-1013f051-80a2-41f1-b893-57ee0644e83b.png)